### DISCUSSION TITLE
Using null in conditionnal flow

### DISCUSSION THREAD

**Discussion by nervusdm** [directus/directus#14671] [Archive]
I can't find a way to use null (or empty) in a conditonnal flow.

Pretty sure it's dead simple but I searched in google.... everywhere.

I want to filter use true / false conditionnal on product_price (if it's null)

Any help would be great

```json
{
    "$last": {
        "product_price": {
            "_eq": "_null"
        }
    }
}
```

**Comment by rijkvanzanten** [comment]
Have you tried `_null: true` instead of `_eq: null`? 🤔

**Reply by nervusdm** [reply]
I used _null :"true",
 thx
